stacker_blueprints/dynamodb.py

```diff
@@ -0,0 +1,152 @@
+from stacker.blueprints.base import Blueprint
```

Comment:
we don't really have a solid standard for this in stacker, but in general I usually try and do imports like:

<native python modules, alphabetical>

<third party modules, alphabetical>

<relative imports>

so this would be:
```

from stacker.blueprints.base import Blueprint
from troposphere import ...

from . import util
```